administrator/components/com_guidedtours/src/Table/TourTable.php

```diff
@@ -109,6 +109,12 @@ public function store($updateNulls = true)
             $this->setTourUid();
         }
 
+        // set autostart
+        // @todo: remove once autostart has been added to the tour form
+        if (empty($this->autostart)) {
```

Comment:
```suggestion
        if (is_null($this->autostart)) {
```
Since 0 is considered empty so this is a better check.